MobalMobal/MobalMobal/Signup/SignupCustomView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SignUpCustomView.swift
+//  MobalMobal
+//
+//  Created by jeongminho on 2021/02/28.
+//
+
+import SnapKit
+import UIKit
+
+class SignupCustomView: UIView {
+    let imageName: String
+    let inputText: String
+    
+    let radiusInputView: UIView = {
+        let view: UIView = UIView()
+        return view
+    }()
+    
+    lazy var imageView: UIImageView = {
+        let imageView: UIImageView = UIImageView(image: UIImage(named: imageName))
+//        imageView.frame.size = CGSize(width: 44, height: 44)
+        return imageView
+    }()
+    
+    lazy var textFieldView: UITextField = {
+        let textField: UITextField = UITextField()
+        textField.font = UIFont(name: "SpoqaHanSansNeo-Bold", size: 15)
+        textField.textColor = .textColor
+        textField.textColor = .white
+        textField.placeholder = inputText
+        return textField
+    }()
+    
+    init(imageName: String, inputText: String) {
+        self.imageName = imageName
+        self.inputText = inputText
+        super.init(frame: .zero)
+        
+        self.addSubview(radiusInputView)
+        radiusInputView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        self.radiusInputView.addSubview(imageView)
+        self.radiusInputView.addSubview(textFieldView)
```

Comment:
```suggestion
       [imageView, textFieldView].forEach { self.radiusInputView.addSubview($0) }
```

이렇게 바꾸면 `radiusInputView`에 subView로 들어가는 친구들이 뭐뭐 있는지 한눈에 알아보기 좋을 것 같아요

Reply:
저번에 같이 얘기했던 것은 기억나는데 저는 그때그때 필요할때 addSubview 하는게 눈에 더 들어와서 일단 이렇게 구현했습니다. Snapkit 조금 더 익숙해지면 리펙토링 할게요

Reply:
@tngusmiso 명확하지 않은 부분도 있는거 같은데 UIView extension으로 좀더 명확한 함수를 하나 만들면 어떨까여??
func addSubViews(_ views: [UIView])이런거??

Reply:
@JeaSungLEE 

이 부분 얘기해보도록 하겠습니당👍